packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -115,7 +115,7 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
     return hexConcat([
       this.factory.address,
       this.factory.interface.encodeFunctionData('deployCounterFactualAccount', [
-        this.owner,
+        ethers.constants.AddressZero,
```

Comment:
Why zero address? i don't get this change.

Reply:
V2 is in progress. I had to give something in abcense of owner address used in V1

Reply:
hmm ok. merging this but in such situation instead of adding wrong values better to comment out the code with TODO